src/commands/env.rs

```diff
@@ -44,6 +46,17 @@ fn make_symlink(config: &FnmConfig) -> std::path::PathBuf {
     temp_dir
 }
 
+fn merge_path(path: &Path) -> Result<String, Error> {
+    let current_path = std::env::var_os("PATH").ok_or(Error::CantReadPath)?;
+    let mut split_paths: HashSet<_> = std::env::split_paths(&current_path)
+        .filter(|p| !p.to_str().unwrap().contains("fnm_multishells"))
+        .collect();
+    split_paths.insert(path.to_path_buf());
+    let new_path =
+        std::env::join_paths(split_paths).map_err(|source| Error::CantJoinPaths { source })?;
```

Comment:
I hope that this works 🤔  what will happen if a Windows user runs Bash in cygwin or other emulators, and eventually call the Windows `.exe` instead of the Linux binary? 🤔 
If we need to join it ourselves by having a function in the trait object, it is going to be a bummer 🐑

Reply:
It's hard to say that it works. It has a lot of problems if the Windows user runs `bash` in Cygwin or other emulators. 

The first problem I just found is the value of the `PATH` variable. The value of the `PATH` variable from Cygwin looks different from the output from fnm. I think the only way to solve this problem is to revert the code to 33c8da27f84c074fc9bc4836d4985b613fbbb269. And join `PATH` by ourselves for every cross-platform shell.

Another problem is the result from `make_symlink()` is based on what OS is using. In the current version of fnm, it still generated Windows path style even if we're using bash. I think this should be a new issue.

Reply:
@Schniz What do u think about the above comment. In addition, I would recommend for the users that use PowerShell in WSL2 to use Linux Binary of `fnm` instead of Windows `.exe`

Reply:
Should we add this recommendation to the document?

Reply:
I think we should 👀

Reply:
good catch 🙌